### ISSUE TITLE
error while deploying via helm

### ISSUE THREAD

**Issue by viveklanjekar** [grafana/mimir#3460]
helm3 install mimir grafana/mimir-distributed --namespace mimir --create-namespace --debug
install.go:159: [debug] Original chart version: ""
install.go:176: [debug] CHART PATH: /Users/vivek.lanjekar/Library/Caches/helm/repository/mimir-distributed-3.3.0.tgz

client.go:108: [debug] creating 1 resource(s)
install.go:140: [debug] CRD podmonitors.monitoring.coreos.com is already present. Skipping.
client.go:108: [debug] creating 1 resource(s)
install.go:140: [debug] CRD probes.monitoring.coreos.com is already present. Skipping.
client.go:108: [debug] creating 1 resource(s)
install.go:140: [debug] CRD servicemonitors.monitoring.coreos.com is already present. Skipping.
client.go:108: [debug] creating 1 resource(s)
install.go:140: [debug] CRD grafanaagents.monitoring.grafana.com is already present. Skipping.
client.go:108: [debug] creating 1 resource(s)
install.go:140: [debug] CRD integrations.monitoring.grafana.com is already present. Skipping.
client.go:108: [debug] creating 1 resource(s)
install.go:140: [debug] CRD logsinstances.monitoring.grafana.com is already present. Skipping.
client.go:108: [debug] creating 1 resource(s)
install.go:140: [debug] CRD metricsinstances.monitoring.grafana.com is already present. Skipping.
client.go:108: [debug] creating 1 resource(s)
install.go:140: [debug] CRD podlogs.monitoring.grafana.com is already present. Skipping.
Error: parse error at (mimir-distributed/templates/validate.yaml:38): unclosed action
helm.go:84: [debug] parse error at (mimir-distributed/templates/validate.yaml:38): unclosed action

**Comment by Logiraptor** [comment]
Hi @viveklanjekar , I'm unable to reproduce this issue locally. I've run the same command with:

```
$ helm version
version.BuildInfo{Version:"v3.9.0", GitCommit:"7ceeda6c585217a19a1131663d8cd1f7d641b2a7", GitTreeState:"clean", GoVersion:"go1.17.5"}

$ helm install tmp-mimir grafana/mimir-distributed --namespace tmp-mimir --create-namespace --debug
install.go:178: [debug] Original chart version: ""
install.go:195: [debug] CHART PATH: /Users/patrickoyarzun/Library/Caches/helm/repository/mimir-distributed-3.3.0.tgz

W1117 15:01:22.364807   31509 warnings.go:70] policy/v1beta1 PodSecurityPolicy is deprecated in v1.21+, unavailable in v1.25+
client.go:128: [debug] creating 1 resource(s)
client.go:128: [debug] creating 46 resource(s)
<....>
```

I don't get any parse error. Can you let us know what the output of `helm3 version` is?

**Comment by bilbof** [comment]
Most likely this is caused by the version of Helm. I saw the same thing with Flux's helm-operator (v1) vs helm-controller (v2).

**Comment by viveklanjekar** [comment]
yes it worked once i updated helm v3.2.4 to helm v3.10.2 , thanks @Logiraptor @bilbof for help
